src/component/Route.js

```diff
@@ -4,7 +4,10 @@ import { NavigationContext } from "../context/NavigationContext";
 function Route({ children, path }) {
   const { currentPath } = useContext(NavigationContext);
 
-  if (currentPath === path) return children;
+  if (path.test(currentPath)) {
```

Comment:
I usually use String [match](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match) instead, so something like `currentPath.match(path)`. Remember to parse `path` to RegExp first, if you want to take my approach.